firefox-mobile/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../.eslintrc",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
+  }
+}
```

Comment:
Nit: new-line at the end of file.

(that's a general Mozilla policy, so I hope it applies here as well! ;))